sites/site-utilities/statics/server/server.js

```diff
@@ -26,7 +26,7 @@ if (process.env.WEBSITE_HOSTNAME.indexOf("create") > -1) {
                 defaultSrc: ["'self'"],
                 fontSrc: ["'self' use.typekit.net static.fast.design c.s-microsoft.com"],
                 frameAncestors: [`'self' ${process.env.FRAME_ANCESTOR_PARTNER}`],
-                imgSrc: ["'self' data: *.fast.design"],
+                imgSrc: ["'self' data: *.fast.design via.placeholder.com"],
```

Comment:
Can we make this even more permissive?

Reply:
I'm not sure what you mean?  What are you suggesting?

Reply:
In the event that someone wants to link off to any site, I can't imagine it would hurt anything if we allowed that. For example let's say I wanted to link to an image on msn or imgur.

Reply:
It's a security violation to host on 3rd party sites, we can only serve assets from Microsoft hosted services.  If you want to make a feature request for this in the future that's great.  But, that's outside the scope of this issue imho.  This PR only unblocks you temporarily until we get an exception for hosting on 3rd party or we create our own placehold.it service to get around this constraint.